tests/integration/security/util/authn/authn_util.go

```diff
@@ -21,13 +21,22 @@ import (
 )
 
 type TestCase struct {
+	Name                string
 	Request             connection.Checker
 	ExpectAuthenticated bool
 }
 
-func getErrorMessage(req connection.Checker, expect, actual string) error {
-	return fmt.Errorf("%s to %s:%s using %s: expected %s, got %s",
-		req.From.Config().Service, req.Options.Target.Config().Service, req.Options.PortName, req.Options.Scheme, expect, actual)
+func (c *TestCase) String() string {
+	want := "deny"
+	if c.ExpectAuthenticated {
+		want = "allow"
+	}
+	return fmt.Sprintf("[%s][%s->%s%s][%s]",
```

Comment:
Please keep "%s to %s:%s using %s: expected %s, got %s",or similar, so that we don't have to decrypt the output message.

Reply:
The port and scheme is the same for every test cases so I think we don't need to encode it, but we should include path, so updated to `"%s to %s%s expected %s"`